poet/poet.py

```diff
@@ -105,75 +109,62 @@ def __getitem__(self, idx):
     def __setitem__(self, idx, val):
         setattr(self, idx, val)
 
-
-
-def get_download_url_from_pip_source_file(module: str, pip_source_file: Path, output_dir: Path) -> str:
-    """
-    Returns the download URL for the pip source distribution.
-    This method will download the pip package from the source distribution. 
-
-    The standard out of this command contains an obfuscated URL and a regular URL that points to a .tar.gz file.
-
-    Arguments:
-        module (str): The name of the module to download.
-        output_dir (str): The directory to download the module to.
-
-    Returns:
-        str: The download URL for the pip source distribution.
-    """
-
-    if not output_dir.is_dir():
-        output_dir = Path(os.getenv("PIP_SOURCE_DIR"))
-    try:
-        output = subprocess.run(shlex.split(f"pip download --dest {output_dir} --no-binary :all: --no-deps {module}"), capture_output=True, text=True)
-    except subprocess.CalledProcessError as cpe:
-        raise PipSourceMetadataException(f"Could not download {module} from pip source file: {cpe.stderr}")
     
-    try:
-        extractor = URLExtract()
-        urls = extractor.find_urls(output.stdout)
-        return [url for url in urls if pip_source_file.name in url][0]
-    except Exception as e:
-        raise PipSourceMetadataException(f"Could not get download URL from pip source file: {e}") from e
-
-def get_checksum_from_pip_source_file(pip_source_file: Path) -> str:
-    """Given the path to a pip source file, return the files checksum.
-
-    Args:
-        pip_source_file (Path): The path to a .tar.gz file containing a pip source distribution.
-
-    Returns:
-        str: The checksum of the pip source file.
-    """
-    if not pip_source_file.exists():
-        raise PipSourceMetadataException("File does not exist: %s" % pip_source_file)
-    
-    return sha256(pip_source_file.read_bytes()).hexdigest()
-
-
-def get_metadata_from_pip_source(pip_source_file: Path) -> PackageMetadata:
-    """Given the path to a pip source file, return a PackageMetadata object.
-
-    Args:
-        pip_source_file (Path): The path to a .tar.gz file containing a pip source distribution.
+    def get_checksums(self) -> str:
```

Comment:
```suggestion
    def get_checksums(self):
```
we are no longer returning a `str` (or anything) from this method